digitalocean/resource_digitalocean_volume_snapshot.go

```diff
@@ -78,6 +82,22 @@ func resourceDigitalOceanVolumeSnapshotCreate(d *schema.ResourceData, meta inter
 	return resourceDigitalOceanVolumeSnapshotRead(d, meta)
 }
 
+func resourceDigitalOceanVolumeSnapshotUpdate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*CombinedConfig).godoClient()
+
+	// godo.VolumeSnapshotResourceType is currently incorrectly set as "volumesnapshot"
+	var VolumeSnapshotResourceType godo.ResourceType = "volume_snapshot"
```

Comment:
As noted in the comment, the current `godo.VolumeSnapshotResourceType` is incorrect so this is a workaround. I opened a [PR](https://github.com/digitalocean/godo/pull/280) to `godo` to fix this value, do we want to wait on that PR before this one goes in?

Reply:
I'll ping the team about cutting a new godo release so that we can get it in all in one PR.

Thanks again!

Reply:
`godo` v1.28.0 has been tagged and released. :tada:

Reply:
great, updated to use v1.28.0 👍